lib/connection.js

```diff
@@ -605,6 +605,30 @@ Connection.prototype.dropCollection = async function dropCollection(collection)
   return this.db.dropCollection(collection);
 };
 
+/**
+ * Helper for MongoDB Node driver's `listCollections()`.
+ * Returns an array of collection objects.
+ *
+ * @method listCollections
+ * @return {Promise<Collection[]>}
+ * @api public
+ */
+
+Connection.prototype.listCollections = async function listCollections() {
+  if (arguments.length >= 2 && typeof arguments[1] === 'function') {
```

Comment:
Nitpick: since we're doing `>=`, I think it's more accurate to check for the last argument instead of argument at position `1`.
```suggestion
  if (arguments.length >= 2 && typeof arguments[arguments.length - 1] === 'function') {
```